bin/cli.class.php

```diff
@@ -220,9 +220,19 @@ private function _bootstrap() {
     if (ord($_SERVER['SCRIPT_NAME']) != 47) {
       $_SERVER['SCRIPT_NAME'] = '/' . $_SERVER['SCRIPT_NAME'];
     }
-
+    $isJoomla = FALSE;
+    if (str_contains(__FILE__, 'administrator')) {
```

Comment:
Per main discussion, I used 'administrator/components/com_civicrm/civicrm/' here as the comparison path, to be a bit more robust and closer to how cv does it.